Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -91,11 +91,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, readonly, nullable)id<STPPaymentMethod> selectedPaymentMethod;
 
 /**
- *  The available payment methods the user can choose between. May be nil.
- */
-@property(nonatomic, readonly, nullable)NSArray<id<STPPaymentMethod>> *paymentMethods;
```

Comment:
Yeah, I'm gonna need to redo a bunch of payment method tuple/exposing stuff after designs are finalized anyway, so I think maybe just merge this for now and redo it later?